FirebaseSessions/Sources/Coordinator.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+///
+/// Main is responsible for coordinating the systems in this SDK involved with sending a Session Start event.
+///
```

Comment:
Xcode supercut: select anywhere in the symbol/function name and do: ⌥ + ⌘ + /  to generate a formatted doc comment.

Reply:
That's awesome - gonna use that from now on